VirtoCommerce.Platform.Web/Scripts/app/security/services/auth.js

```diff
@@ -119,7 +119,7 @@ angular.module('platformWebApp')
 
                         // if we have no scope means we have permission
                         var retVal = !_.some(authContext.permissions, function (item) {
-                            return item === scopeName || item.indexOf(scopeName + ':') > -1;
+                            return item === scopeName || !item.indexOf(scopeName + ':');
```

Comment:
As i understand, thus we want to check if string starts with another. I think 
`item.indexOf(scopeName + ':') === 0`
is better to understand than 
`!item.indexOf(scopeName + ':')` 
in that case.

Reply:
fixed